### PR TITLE
enforce trailing commas in subscript tuples and slices

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: flake8_commas/_base.py [hunk 1]
```diff
@@ -144,6 +147,7 @@ def simple_tokens(tokens):
 
 ERRORS = {
     True: ('C812', 'missing trailing comma'),
+    SUBSCRIPT_TUPLE: ('C812', 'missing trailing comma'),
```
Comment (graingert) [inline flake8_commas/_base.py]: conflating the ERROR dict and the comma context is getting horrible...

File: test/data/multiline_subscript_slice.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+multiline_index_access[
+    lambda fine,
+    fine,
+    fine: (0,)
+    :
+    lambda fine,
+    fine,
+    fine: (0,),
+    "not fine"
```
Comment (graingert) [inline test/data/multiline_subscript_slice.py]: wow, this is some horrible code, but by jove does it need a comma right here.

**Comment by graingert** [comment]
@PeterJCLaw can you check this still works on your codebase?

**Comment by PeterJCLaw** [comment]
Sure. (I might also submit a PR to clarify some of the test cases)

**Comment by PeterJCLaw** [comment]
This seems fine on my codebase, thanks.

**Pull request merged**